docker-compose.yml

```diff
@@ -16,18 +16,34 @@ services:
             - AXIS_SERVER=go,node,java,python
             - AXIS_TRANSPORT=http,tchannel
             - AXIS_ENCODING=raw,json,thrift
-            - AXIS_ERRORS=go,node
-            - AXIS_GAUNTLET=go
 
             - BEHAVIOR_RAW=client,server,transport
             - BEHAVIOR_JSON=client,server,transport
             - BEHAVIOR_THRIFT=client,server,transport
             - BEHAVIOR_HEADERS=client,server,transport,encoding
-            - BEHAVIOR_ERRORS=errors,server
+            - BEHAVIOR_OUTBOUNDTTL=client,server,transport
+            # TODO INBOUND TTL
+
             - BEHAVIOR_TCHCLIENT=client,server,encoding
             - BEHAVIOR_TCHSERVER=client,server,encoding
+            # TODO HTTP CLIENT
+            # TODO HTTP SERVER
+
+            # ERRORS HTTP OUT
+            - BEHAVIOR_ERRORS=errorshttpout,server
+            - AXIS_ERRORSHTTPOUT=go,node
+
+            # ERRORS TCH OUT
+            - BEHAVIOR_ERRORSTCHOUT=errorstchout,server
+            - AXIS_ERRORSTCHOUT=go
+
+            # TODO ERRORS HTTP IN
+
+            # TODO ERRORS TCH IN
+
+            # THRIFT GAUNTLET
             - BEHAVIOR_THRIFTGAUNTLET=gauntlet,server,transport
-            - BEHAVIOR_OUTBOUNDTTL=client,server,transport
+            - AXIS_GAUNTLET=go
```

Comment:
staring at this file for awhile now - honestly cant tell if it's easier or harder to read now :)